src/blob_gc.h

```diff
@@ -13,7 +13,7 @@ namespace titandb {
 class BlobGC {
  public:
   BlobGC(std::vector<BlobFileMeta*>&& blob_files,
-         TitanCFOptions&& _titan_cf_options);
+         TitanCFOptions&& _titan_cf_options, bool need_trigger_next = false);
```

Comment:
seems there's no point to have `need_trigger_next` being optional. I prefer to avoid optional params because the caller may forget to pass them sometimes.